testing/src/test/java/org/oppia/android/testing/FakePerformanceMetricUtilsTest.kt

```diff
@@ -0,0 +1,164 @@
+package org.oppia.android.testing
+
+import android.app.Application
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import dagger.BindsInstance
+import dagger.Component
+import dagger.Module
+import dagger.Provides
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.android.app.model.OppiaMetricLog
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+private const val TEST_APK_SIZE = Long.MAX_VALUE
+private const val TEST_STORAGE_USAGE = Long.MAX_VALUE
+private const val TEST_TOTAL_PSS = Long.MAX_VALUE
+private const val TEST_BYTES_SENT = Long.MAX_VALUE
+private const val TEST_BYTES_RECEIVED = Long.MAX_VALUE
```

Comment:
Perhaps ``DEFAULT_TEST_APK_SIZE`` would be clearer? Ditto for the others.

Reply:
done.